testsuite/forge-cli/src/main.rs

```diff
@@ -185,6 +185,7 @@ fn main() -> Result<()> {
     let args = Args::from_args();
     let duration = Duration::from_secs(args.duration_secs as u64);
     let suite_name: &str = args.suite.as_ref();
+    let duration = Duration::from_secs(3600);
```

Comment:
Any reason we're hard coding this? Or was it just for testing?

Reply:
yeah just testing